### PR TITLE
fix: Lazy load CSS also

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java [hunk 1]
```diff
@@ -271,20 +286,17 @@ String resolveGeneratedModule(String module) {
         return FrontendUtils.FRONTEND_GENERATED_FLOW_IMPORT_PATH + module;
     }
 
-    protected List<String> getCssLines(Map<ChunkInfo, List<CssData>> css) {
+    protected List<String> getCssLines(List<CssData> css) {
```
Comment (caalador) [inline flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:289]: The index for cssLine should be a rising global as now we will get 0 for each css key

**Review by caalador — CHANGES_REQUESTED** [review]

File: flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java (hunk 1 above)
Comment (caalador) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:289]: This should now be fine in master, but needs conflict resolution.

Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:289]: Is it fine to have
```
import $cssFromFile_0 from 'Frontend/foo.css?inline';
injectGlobalCss($cssFromFile_0.toString(), 'CSSImport end', document);
```

in multiple JS files?

Comment (caalador) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:289]: I think it should be fine to have the same import identifier in different files, but ofc not having the same numbering might be better.

Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:289]: The question is not really the numbering but does `injectGlobalCss` handle deduplication and does having the import in multiple files cause overhead or not? I think the answer is "yes" and "does not cause overhead" but maybe it needs to be checked

Comment (caalador) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:289]: Aa. The css is checked for if it has been injected with the exact same hash when target is document so it handles duplication. So overhead is duplicate data transmitted/stored in bundle and hash generation.

Comment (Artur-) [reply flow-server/src/main/java/com/vaadin/flow/server/frontend/AbstractUpdateImports.java:302]: Ok, then this version could work

**Review by caalador — APPROVED** [review]

**Pull request merged**